openapi/components/schemas/ReportJournal.yaml

```
@@ -0,0 +1,27 @@
+type: array
+items:
+  type: object
+  properties:
+    aggregationValue:
+      description: The aggregation field value.
+      type: string
+    bookedMonth:
+      description: Revenue booked month.
+      type: string
+      pattern: '^\d{4}-\d{2}$'
+      example: 2022-02
+    bookedAmount:
+      description: The revenue amount booked for recognition.
+      type: number
+      format: double
+      example: 25.0
+    recognizedAmount:
+      description: The revenue amount recognized.
+      type: number
+      format: double
+      example: 20.0
+    remainingAmount:
+      description: The revenue amount remaining to be recognized.
```

nitpick: We should use the adjective before the noun

The placement of "remaining" (adjective) made it a little confusing

```suggestion
      description: Remaining revenue amount to be recognized.
```